backend/pong/consumers.py

```diff
@@ -1,19 +1,17 @@
 # pong/consumers.py
-from django.db import transaction
 import json
 import asyncio
+from asgiref.sync import sync_to_async
 from channels.generic.websocket import AsyncWebsocketConsumer
 from .game import PongStatus
 from .constants import *
 from channels.db import database_sync_to_async
 from Tokens.models import MatchToken
 from Match.models import Match
-from Match.matchState import MatchState
```

Comment:
The imports are necessary. I recommend copy-pasting the stuff you've added in this pr to a pulled file from main. Some of the changes in this file tells me that something went wrong with git or something of the sorts.

Reply:
There is some weirdness with the commits to this file for sure.... has there been a sqash merge or something?

Reply:
Still shouldn't be possible at least not squashing PRs onto the main. What people do on their branches is up to them (so maybe?)